R/remove_variables.R

```diff
@@ -0,0 +1,60 @@
+# remove variables from a draws object
+remove_variables <- function(x, ...) {
```

Comment:
The code for these methods is pretty simple, but maybe still worth adding some simple tests for these?

Reply:
Added some unit tests.